### DISCUSSION TITLE
How to push custom metrics to Statsd / Statsd exporter

### DISCUSSION THREAD

**Discussion by andormarkus** [apache/airflow#37245] [Q&A]
Hi All,

We would like to push custom metrics to Statsd / Statsd exporter so the values will show up in Prometheus.

I have tried the following code based on `base_executor.py` code  but metrics are not showing up

```python
from airflow.stats import Stats

Stats.incr("my_custom_vaue", 100)
```

Thank you so much for you support,
Andor

**Comment by andresitorusz** [comment]
Where did you call the base_executor.py? You need to make it as a function I guess then call the function, like:

```python
from airflow.stats import Stats

def push_custom_metrics(**context):
    try:
        Stats.incr("my_custom_vaue", 100)
    except:
        ...
```

**Reply by shubhamqua** [reply]
Can you tell me where can i call this function push_custom_metrics  from the dag.?
